internal/controller/hubble/hubble.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2025 Illumio, Inc. All Rights Reserved.
+
+package hubble
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+
+	exp_credentials "github.com/illumio/cloud-operator/internal/tls"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/credentials"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	hubbleRelayExpectedServerName        = "ui.hubble-relay.cilium.io"
+	ciliumHubbleRelayServiceName  string = "hubble-relay"
```

Comment:
Let's be consistent: "Hubble" -> "Cilium Hubble" everywhere.
```suggestion
	ciliumHubbleRelayExpectedServerName        = "ui.hubble-relay.cilium.io"
	ciliumHubbleRelayServiceName  string = "hubble-relay"
```